service/gcs/runtime/runc/runc_test.go

```diff
@@ -182,7 +174,7 @@ var _ = Describe("runC", func() {
 			c  runtime.Container
 		)
 		JustBeforeEach(func() {
-			c, err = rtime.CreateContainer(id, bundle, createAllStdioOptions)
+			c, err = rtime.CreateContainer(id, bundle, &stdio.ConnectionSet{})
```

Comment:
I see, you mean so that we can exercise this code in the tests, right? It doesn't look like any tests actually validate stdio behavior today, but I might be missing something.